modules/querier/querier.go

```diff
@@ -254,6 +254,11 @@ func (q *Querier) forGivenIngesters(ctx context.Context, replicationSet ring.Rep
 }
 
 func (q *Querier) Search(ctx context.Context, req *tempopb.SearchRequest) (*tempopb.SearchResponse, error) {
+	_, err := user.ExtractOrgID(ctx)
+	if err != nil {
+		return nil, errors.Wrap(err, "error extracting org id in Querier.Search")
+	}
```

Comment:
re-added this because we may want to check orgID at the queriers before firing search queries for all ingesters.